CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## 0.8.1 / 2022-10-21
```

Comment:
@SuperQ I believe this should go under a new # 0.9.0 paragraph

Reply:
Whups, thanks